WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Free Trial/FreeTrialSurvey/FreeTrialSurveyViewModelTests.swift

```diff
@@ -188,4 +189,63 @@ final class FreeTrialSurveyViewModelTests: XCTestCase {
         // Then
         XCTAssertTrue(onCloseFired)
     }
+
+    // MARK: Local notification after three days
+
+    func test_threeDaysAfterStillExploring_local_notification_is_scheduled_if_submitted_answer_is_stillExploring() throws {
+        // Given
+        let sessionManager = SessionManager.testingInstance
+        let stores = MockStoresManager(sessionManager: sessionManager)
```

Comment:
Nit: we can send the default store to `MockStoresManager` without having to set it to session manager.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10331/commits/64abe52544ae91c013bedbb1472f0f9c49af76bd